src/atoms/weavemol.jl

```diff
@@ -1,13 +1,16 @@
 # some commentary
 
 # docstring...
+
 mutable struct WeaveMol <: AbstractAtoms
     smiles::String
     elements::Vector{String}
-    atom_features::Vector{SomethingOrOther} # I need to look more carefully to figure this out, heh
-    pair_features::Vector{SomethingOrOther}
+    atom_features::Vector{SomethingOrOther} # Details TBD
```

Comment:
so this will be encoded atom features, meaning type probably ends up as a matrix (stacked vectors, one for each atom)